go.mod

```diff
@@ -252,7 +252,11 @@ require (
 
 replace (
 	github.com/google/cel-go => github.com/google/cel-go v0.17.7
+	k8s.io/client-go => k8s.io/client-go v0.29.0
```

Comment:
We've been trying to remove as much as the replaces as we can. Can these be removed? Our dependencies have been updated to k8s v0.29

Reply:
I can remove `k8s.io/client-go` but if I remove `k8s.cri-api`,`k8s.io/csi-translation-lib`or `k8s.io/mount-utils` I get an error `invalid version: unknown revision v0.0.0`